TACC/Core-Portal
# TV3-76: Execution systems onboarding

## Overview

Genericize the onboarding step for tapis system access

## Related

* [TV3-76](https://jira.tacc.utexas.edu/browse/TV3-76)


## Testing

1. login while on VPN
2. Confirm the onboarding step sets up access
3. You can confirm access by re-running the onboarding step and seeing if the logs tell you you already have access for that system

## UI Photos
<img width="1348" alt="Screen Shot 2022-10-20 at 12 26 25 PM" src="https://user-images.githubusercontent.com/20326896/197028692-76a35046-424f-4831-a6f9-8b3fbcd3af1e.png">


## Notes
- We still need a fallback for systems that don't work with the keys service
